libcontainerd/client_linux.go

```diff
@@ -464,7 +464,7 @@ func (clnt *client) Restore(containerID string, attachStdio StdioCallback, optio
 	cont, err := clnt.getContainerdContainer(containerID)
 	// Get its last event
 	ev, eerr := clnt.getContainerLastEvent(containerID)
-	if err != nil || cont.Status == "Stopped" {
+	if err != nil || cont.Status == "stopped" {
```

Comment:
@mlaventure ,Thanks, I have tried to update vendor with hack/vendor.sh, but failed because of network problems. feel free to take this pr over  :)